File: trunk/emsi/asp-net-app/WebEMSOF/component/biz/Class_biz_payment_proof_methods.pas
unit Class_biz_payment_proof_methods;

interface

uses
  Class_db_payment_proof_methods;

type
  TClass_biz_payment_proof_methods = class
  private
    db_payment_proof_methods: TClass_db_payment_proof_methods;
  public
    constructor Create;
    procedure BindDropDownList(target: system.object);
  end;

implementation

constructor TClass_biz_payment_proof_methods.Create;
begin
  inherited Create;
  // TODO: Add any constructor code here
  db_payment_proof_methods := TClass_db_payment_proof_methods.Create;
end;

procedure TClass_biz_payment_proof_methods.BindDropDownList(target: system.object);
begin
  db_payment_proof_methods.BindDropDownList(target);
end;

end.

File: asp-net-app/WebEMSOF/component/biz/Class_biz_payment_proof_methods.cs
using System;
using Class_db_payment_proof_methods;
namespace Class_biz_payment_proof_methods
{
    public class TClass_biz_payment_proof_methods
    {
        private TClass_db_payment_proof_methods db_payment_proof_methods = null;
        //Constructor  Create()
        public TClass_biz_payment_proof_methods() : base()
        {
            // TODO: Add any constructor code here
            db_payment_proof_methods = new TClass_db_payment_proof_methods();
        }
        public void BindListControl(object target)
        {
            db_payment_proof_methods.BindListControl(target);
        }

    } // end TClass_biz_payment_proof_methods

}
